charts/tekton/Chart.yaml

```diff
@@ -8,13 +8,13 @@ type: application
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
 appVersion: "v0.28.2"
-dashboardVersion: "v0.21.0"
-triggersVersions: "v0.16.0"
+# dashboardVersion: "v0.21.0"
+# triggersVersions: "v0.16.0"
```

Comment:
Why commented out? If not needed - just remove, or add some TODO comment

Reply:
agreed

Reply:
let's keep it not commented so we know what version we run

Reply:
uncommented. On one of previous builds this was breaking chart validation in GitHub action, that's why I commented it.